### ISSUE TITLE
Add option to disable showing clipped output of user's program

### ISSUE THREAD

**Issue by ArcCha** [DMOJ/site#677]
I've been using dmoj with my collegues (@apardyl and others) in our student's association to host algorithmic challenges. It works pretty well, we're all thankful for the project!

After around 4 weeks, we stated that it is not so uncommon for our users to try to leak test data using the "clipped output" feature on submission page. To prevent that from happening, we need to develop much more complicated test data that we actually need in order to grade submissions properly. It's particularly annoying when we want to have small test cases that test edge cases, which should be discovered by the user himself. I understand that this option is tempting, but it diminishes the education value of the challenge itself.

To sum up, it would be nice to have an option to disable showing clipped output from users program per problem.

**Comment by quantum5** [comment]
Yes, set `output_prefix_length` in init.yml to 0. This can be done for the entire problem, or for individual batches/test cases by putting it in the relevant scope.

**Comment by ArcCha** [comment]
@quantum5 it is not possible to set this using admin interface, or am I missing something?

**Comment by quantum5** [comment]
You mean the online test data editor? It's `Output prefix length:`.

**Comment by ArcCha** [comment]
Oh, I see, it's a column hidden by default. Sorry for the trouble and thanks!

**Comment by quantum5** [comment]
Isn't there a global setting on the top?

![default](https://user-images.githubusercontent.com/461885/39116962-9526bb0e-469a-11e8-8f49-0cff9c388a97.png)

**Comment by ArcCha** [comment]
It is, but I thought it would be nice to leave it as it is for pretest cases.
